### ISSUE TITLE
api model and dataset

### ISSUE THREAD

**Issue by KevinfromTJ** [IDEA-Research/Grounding-DINO-1.5-API#39]
Hello,
Thank you for this excellent work !
I'm wondering whether the model called by Grounding-DINO-1.5 API is finetuned on COCO or Grounding datasets like refcoco ?

**Comment by rentainhe** [comment]
> Hello, Thank you for this excellent work ! I'm wondering whether the model called by Grounding-DINO-1.5 API is finetuned on COCO or Grounding datasets like refcoco ?

No, our API model was not trained on COCO images (which are included in COCO/LVIS/RefCOCO datasets)
